extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -148,7 +149,7 @@ export class SqlDatabaseTree extends AssessmentDialogComponent {
 				rowNumber = rowNumber + 1;
 			});
 
-			dbList.forEach((value) => {
+			dbList.filter(db => db !== 'master' && db !== 'model' && db !== 'tempdb' && db !== 'msdb').forEach((value) => {
```

Comment:
Just have an array of excluded databases (probably as a static class variable).
excludedDatabses = ['master', 'tempdb', 'master', 'model'];
and do 
dbList.filter (db => ! excludedDatabses.includes(db));

Reply:
It will be easier to add or remove databases later.